DFC.ServiceTaxonomy.GraphVisualiser/Controllers/VisualiseController.cs

```diff
@@ -43,41 +55,86 @@ namespace DFC.ServiceTaxonomy.GraphVisualiser.Controllers
     public class VisualiseController : Controller
     {
         private readonly IGraphCluster _neoGraphCluster;
+        private readonly IContentManager _contentManager;
         private readonly IContentDefinitionManager _contentDefinitionManager;
         private readonly INeo4JToOwlGeneratorService _neo4JToOwlGeneratorService;
+        private readonly ISyncNameProvider _graphSyncHelper;
         private readonly IOrchardToOwlGeneratorService _orchardToOwlGeneratorService;
-
+        private readonly IPublishedContentItemVersion _publishedContentItemVersion;
+        private readonly IPreviewContentItemVersion _previewContentItemVersion;
+        private readonly IEnumerable<IContentPartGraphSyncer> _partSyncers;
+        private readonly IEnumerable<IContentFieldGraphSyncer> _fieldSyncers;
+        private readonly IServiceProvider _serviceProvider;
+        private readonly IDescribeContentItemHelper _describeContentItemHelper;
+        private readonly IContentItemGraphSyncer _contentItemGraphSyncer;
```

Comment:
`_contentItemGraphSyncer` isn't used.
I think it needs to call `AddRelationship` on at least the `ContentItemGraphSyncer`, and I would've thought it would also need to ask `TaxonomyTermContentItemGraphSyncer` to properly handle visualising taxonomies (although that would depend on the implementation).
Have a look at where we inject `IEnumerable<IContentItemGraphSyncer>` for examples.